xcm/xcm-executor/src/lib.rs

```diff
@@ -234,7 +234,6 @@ impl<Config: config::Config> XcmExecutor<Config> {
 				None
 			},
 			(origin, Xcm::RelayedFrom { who, message }) => {
-				ensure!(who.is_interior(), XcmError::EscalationOfPrivilege);
```

Comment:
explain?

Reply:
Oh wow, I think the original intent here is to recognize that it's absolutely impossible to create a `Junction::Parent` with the new `MultiLocation` changes, but now thinking of it more closely, we really need to check that `who` has a parent count of 0.